practices/practice-one/src/styles/components/button.css

```diff
@@ -0,0 +1,80 @@
+/* 
+    add to late
+*/
+
+.btn {
+    background-color: transparent;
+    cursor: pointer;
+}
+
+.btn-tick::after {
+    position: absolute;
+    content: '';
+    height: 10px;
+    width: 3px;
+    background-color: var(--cool-color);
+    left: 50%;
+    top: 50%;
+    transform: rotate(-45deg) translate(-50%, -50%);
+}
+
+.btn-tick::before {
+    content: '';
+    height: 18px;
+    width: 3px;
+    background-color: var(--cool-color);
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: rotate(45deg) translate(-50%, -50%);
+}
+
+.btn-checkbox {
```

Comment:
For these buttons, so hard to re-use.

We should defined as basically type of button (default, primary, secondary)
In case you want to custom any button, you can add more class to custom it